lib/components/Editor/CustomExtensions/useCustomExtensions.js

```diff
@@ -35,12 +35,11 @@ const useCustomExtensions = ({
   onSubmit,
   keyboardShortcuts,
   uploadEndpoint,
-  config,
   isVideoEmbedActive,
   setMediaUploader,
   setIsEmbedModalOpen,
 }) => {
-  const customConfig = mergeLeft(config, DEFAULT_CONFIG);
+  const customConfig = DEFAULT_CONFIG;
```

Comment:
We no longer need this variable since DEFAULT_CONFIG does not provide any extra information. Please remove all the usages and refactor the extensions as:

```diff
- ExtensionName.configure(customConfig.underline),
+ ExtensionName,
```